filebeat/input/filestream/internal/input-logfile/input.go

```diff
@@ -64,15 +64,13 @@ func (inp *managedInput) Run(
 
 	hg := &defaultHarvesterGroup{
 		pipeline:     pipeline,
-		readers:      newReaderGroupWithLimit(inp.harvesterLimit),
+		readers:      newReaderGroup(),
 		cleanTimeout: inp.cleanTimeout,
 		harvester:    inp.harvester,
 		store:        groupStore,
 		ackCH:        inp.ackCH,
 		identifier:   inp.sourceIdentifier,
-		tg: unison.TaskGroup{
-			OnQuit: unison.ContinueOnErrors, // harvester should keep running if a single harvester errored
-		},
+		tg:           task.NewGroup(inp.harvesterLimit, 2*inp.harvesterLimit),
```

Comment:
Just wondering: is there some logic to choosing `maxErrors`, the 2nd argument to `task.NewGroup`, as `2 x inp.harvesterLimit`?

Reply:
Based on our conversation off-PR, you told me you're getting rid of the idea of the ring buffer for errors altogether. So this thread is now irrelevant.